src/routes/_index/component/avatar/+page.marko

```diff
@@ -306,86 +346,84 @@
             </div>
         </div>
     </div>
-    <highlight-code
-        type="html"
-        >
+    <highlight-code type="html">
 
-    <div
-        class="avatar avatar--32"
-        role="img"
-        aria-label="Profile picture - Elizabeth"
-    >
-        <img
-            src="https://ir.ebaystatic.com/cr/v/c01/skin/docs/dog_profile_optimized.jpg"
-            alt=""
+        <div
```

Comment:
I'm not sure, but having a `<div role="image">` with an `<img>` inside may be an `a11y` issue. I would think whatever is inside `role="image"` should be `aria-hidden="true"` just like the `svg` variation above. @saiponnada , @ianmcburnie , what do you think?

Reply:
Regardless, this is a completely separate issue to this current PR. If we want to change the structure that should be done in a different issue.

Reply:
https://github.com/eBay/skin/pull/2226 FYI, this was done intentional by Ian.